docs/guide/generators.md

```diff
@@ -82,6 +82,25 @@ bin/rails generate component Example title --stimulus
 
 To always generate a Stimulus controller, set `config.view_component.generate_stimulus_controller = true`.
 
+### Generate [locale files](/guide/translations.html)
+
+```console
+bin/rails generate component Example title --locales
+
+      create  app/components/example_component.rb
+      invoke  test_unit
+      create    test/components/example_component_test.rb
+      invoke  locales
+      create    app/components/example_component.en.yml
+        gsub    app/components/example_component.en.yml
+      create    app/components/example_component.fr.yml
+        gsub    app/components/example_component.fr.yml
+      invoke  erb
+      create    app/components/example_component.html.erb
+```
+
+To always generate locale files, set `config.view_component.generate_locales = true`. Make sure I18n.available_locales is defined in your project, so that only the necessary locale files are created when generating a component.
```

Comment:
```suggestion
To always generate locale files, set `config.view_component.generate_locales = true`. Make sure `I18n.available_locales` is defined in your project, so that only the necessary locale files are created when generating a component.
```